pkgs/applications/office/zanshin/default.nix

```
@@ -1,21 +1,19 @@
 {
   mkDerivation, lib,
-  fetchFromGitHub,
+  fetchurl,
   extra-cmake-modules,
   qtbase, boost,
   akonadi-calendar, akonadi-notes, akonadi-search, kidentitymanagement, kontactinterface, kldap,
-  krunner, kwallet
+  krunner, kwallet, kcalendarcore
 }:
 
-mkDerivation {
+mkDerivation rec {
   pname = "zanshin";
-  version = "2019-07-28";
+  version = "0.5.71";
 
-  src = fetchFromGitHub {
-    owner = "KDE";
-    repo = "zanshin";
-    rev = "a8c223e745ed7e6aa3dd3cb0786a625a5c54e378";
-    sha256 = "0jglwh30x7qrl41n3dhawn4c25dmrzscpvcajhgb6fwcl4w8cgfm";
+  src = fetchurl {
+    url = "https://download.kde.org/stable/${pname}/${pname}-${version}.tar.xz";
```

It would be nice to use the kde fetchurl mirror so that we aren't relying solely upon download.kde.org for mirror resolution.
```suggestion
    url = "mirror://kde/stable/${pname}/${pname}-${version}.tar.xz";
```

---

Oh yes, thank you.  Updated as suggested, tested fetching still works :).